fl4health/utils/functions.py

```diff
@@ -41,3 +45,65 @@ def backward(ctx: torch.Any, grad_output: torch.Tensor) -> torch.Tensor:  # type
 
 def sigmoid_inverse(x: torch.Tensor) -> torch.Tensor:
     return -torch.log(1 / x - 1)
+
+
+def select_random_element(array: np.ndarray) -> float:
+    """
+    Helper function that simply selects a random element from the provided array using np.random.choice.
+
+    Args:
+        array (np.ndarray): Array from which a single element is randomly selected
+
+    Returns:
+        float: randomly selected element value.
+    """
+    indices = tuple(np.random.choice(size) for size in array.shape)
+    return array[indices]
+
+
+def pseudo_sort_function(client_result: Tuple[ClientProxy, NDArrays, int]) -> float:
```

Comment:
nit: This isn't really sorting anything is it? Maybe pseudo_norm or something getting at this is getting a score as opposed to sorting?

Reply:
Changed to `pseudo_sort_scoring_function` to clarify that this is responsible for scoring not sorting specifically.